### PR TITLE
Add a --stage-two to big_red_button.py

### PR DESCRIPTION
**Pull request by johnmccutchan**
@abarth

### CODE REVIEW

File: sky/tools/big_red_button.py [hunk 1]
```diff
@@ -114,13 +116,15 @@ def main():
 
     run(sky_engine_root, ['cp', 'AUTHORS', 'LICENSE', sky_package_root])
 
-    upload_artifacts(android_dist_root, 'android-arm', commit_hash)
-    upload_artifacts(linux_dist_root, 'linux-x64', commit_hash)
+    if args.stage_two:
```
Comment (abarth) [inline sky/tools/big_red_button.py]: These should go in stage 1

**Comment by abarth** [comment]
LGTM w/ note above.

**Comment by johnmccutchan** [comment]
@abarth done.

**Comment by eseidelGoogle** [comment]
We may need to rename this to the big red rocket. :)

**Pull request merged**